### ISSUE TITLE
HMRC

### ISSUE THREAD

**Issue by terenn** [authenticatorplus/logos#6]
Please provide below info, all are mandatory

1. https://www.gov.uk/government/organisations/hm-revenue-customs
2. HMRC - HMRC Online
3. ![hmrc_crest](https://cloud.githubusercontent.com/assets/496244/24715619/feaa3f5a-1a23-11e7-8306-5da6bcd18e4f.png)

**Comment by MufriA** [comment]
Merged to the logo repo, thanks for the contribution.

**Comment by Lantizia** [comment]
@terenn any chance you can share some information on how to set this up within HMRC Online... I don't see any option other than phone-based 2fa.

**Comment by Lantizia** [comment]
Also wouldn't this be a login for Government Gateway and not specifically just for HMRC?

**Comment by terenn** [comment]
@Lantizia That is a good question. I don't remember exactly, but I think the registration was here https://www.tax.service.gov.uk/government-gateway-registration-frontend?accountType=individual&continue=%2Fpersonal-account

What I've done afterwards I don't remember, sorry. But I do know that, at least for me, the Gov.uk accounts for different services are not shared, i.e. I have to use one account to access DVLA stuff and another one for HMRC. Maybe I registered before they merged them, who knows.

**Comment by Lantizia** [comment]
Yes I've been able to set this up now on my personal HRMC account, earlier I was trying to find the option when I was logged in as a business - doesn't exist there (must be phasing it in I guess).

Small bug...

The issuer name of "HMRC - HMRC Online" is wrong, it's actually just "HMRC" (I've checked the otpauth:// URL)... the "HMRC Online" bit appears after the colon (as though it's the username, which is just stupid).  

This explains why the logo doesn't automatically populate when you scan the QR code.
